src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig

```diff
@@ -82,6 +82,8 @@
 
     <h2>Translation Messages</h2>
 
+    {% block panelContentHead %}{% endblock %}
```

Comment:
I don't like this block name because the panel content starts just after `{% block panelContent %}`, so this can't be the head of the panel content. What if we rename {% block panelContentHead %} as `{% block before_translation_messages %}` and `{% block panelContentFooter %}` as `{% block after_translation_messages %}`